### PR TITLE
add azuredisk v1.1.0 image

### PR DESCRIPTION
**Pull request by nilo19**
add azuredisk v1.1.0 image

### CODE REVIEW

File: vhdbuilder/packer/test/vhd-content-test.ps1 [hunk 1]
```diff
@@ -183,6 +183,7 @@ function Test-ImagesPulled {
                 "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v0.0.19",
                 "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:0.0.12",
                 "mcr.microsoft.com/k8s/csi/azuredisk-csi:v1.0.0",
+                "mcr.microsoft.com/k8s/csi/azuredisk-csi:v1.1.0",
                 "mcr.microsoft.com/k8s/csi/azurefile-csi:v1.0.0")
```
Comment (adilliadil) [inline vhdbuilder/packer/test/vhd-content-test.ps1:187]: why do we have this duplicate

**Review by adilliadil — APPROVED** [review]
lgtm

File: vhdbuilder/packer/test/vhd-content-test.ps1 (hunk 1 above)
Comment (nilo19) [reply vhdbuilder/packer/test/vhd-content-test.ps1:187]: there is a problem in integrating azuredisk 1.0.0 with aks

File: vhdbuilder/packer/install-dependencies.sh [hunk 2]
```diff
@@ -648,6 +648,7 @@ AZUREDISK_CSI_VERSIONS="
 0.7.0
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:648]: do we still use this? if not we should remove it to preserve disk space

Comment (nilo19) [reply vhdbuilder/packer/install-dependencies.sh:648]: I think it's not needed but @andyzhangx could correct me.

Comment (andyzhangx) [reply vhdbuilder/packer/install-dependencies.sh:648]: yes, let's move this

**Review by andyzhangx — APPROVED** [review]
/lgtm

**Pull request merged**